src/archiefvernietigingscomponent/destruction/templates/destruction/recordmanager_list.html

```diff
@@ -24,19 +24,24 @@ <h1 class="title">Mijn vernietigingslijsten</h1>
 
                 <span class="destruction-list-preview__indicators">
 
-                    <span class="destruction-list-preview__assigned">
-                        {% if object.assignee == request.user %}
-                            <i class="material-icons" title="{% trans 'You are the current assignee!' %}">assignment_ind</i>
+                    <span class="destruction-list-preview__label">
+                        {{ object.list_state.label }}
+                        {% if object.list_state.value == "in_progress" %}
+                            ({{ object.completed_reviewers }}/{{ object.total_reviewers }} {% trans "reviewers " %})
                         {% endif %}
                     </span>
 
-                    <span class="destruction-list-preview__status" title="{{ object.get_status_display }}">
-                        {% if object.status == "in_progress" %}
+                    <span class="destruction-list-preview__status">
+                        {% if object.list_state.value == "in_progress" %}
```

Comment:
I think using both material icons and state label is excessive - one of them should be enough
I'd prefer icons with text states as tooltips. Otherwise, when you have a lot of finished destruction lists, you'll have "finished" word repeated a lot of times on the page

But perhaps we should ask product owners

Reply:
After discussion with the product owners, we should keep the labels, but add translations!